db/migrate/20230126164114_create_faculty_member.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFacultyMember < ActiveRecord::Migration[6.0]
+  def change
+    create_table :faculty_members do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :webaccess_id, null: false, default: ""
+    end
+    add_foreign_key :faculty_members, :committee_members, name: :faculty_members_committee_id_fk
```

Comment:
We can add the foreign key to this migration, but we'll have to change the arguments to `add_foreign_key` here.  It should look like this: 
`add_foreign_key :committee_members, :faculty_members, name: :committee_members_faculty_member_id_fk`

The first argument is the name of the table that `belongs_to` the other table.  The second argument is the other table.  The name should just reflect that relationship.

We'll also want a unique index like this below line 8:

`add_index :faculty_members, :webaccess_id, unique: true`

This will make sure we don't end up with records that have the same access ids.